### DISCUSSION TITLE
What about Events?

### DISCUSSION THREAD

**Discussion by runxc1** [bluesky-social/atproto#864] [Bluesky Lexicons]
What about something like an Event that you want to create and broadcast either to friends or to the public.    The content is similar to a post but has additional data such as place, time, etc.   Does the AT protocol allow anything other than a simple post?

**Comment by dholms** [comment]
AT protocol does allow it, but we have not mapped out the schemas & interaction model for events yet.

Not super high priority - but likely on our roadmap at some point. Jay, the CEO, actually founded an events-based social network before Bluesky!

**Comment by runxc1** [comment]
I'd be very interested in the roadmap.    One because I've found that planning private events is why most friends are still on FB and I'd like to not support the company and two because I work at one of the largest event platform companies.   We have a calendar on thousands of large media sites and allowing users to share events with friends or social media sites is always a request.
